control/veda_val.py

```diff
@@ -9,7 +9,10 @@
 import urllib3
 import ast
 import json
+import datetime
 
+from django.core.cache import cache
+from edx_rest_api_client.client import EdxRestApiClient
```

Comment:
`EdxRestApiClient` is actually deprecated.
See https://github.com/edx/edx-video-worker/pull/55#discussion_r328193241

Reply:
@doctoryes I have updated to use OAuthAPIClient, please review when you get a chance.  Thanks!